qiita_db/study.py

```diff
@@ -617,11 +635,14 @@ def investigation(self):
         -------
         Investigation id
         """
-        conn_handler = SQLConnectionHandler()
-        sql = ("SELECT investigation_id FROM qiita.investigation_study WHERE "
-               "study_id = %s")
-        inv = conn_handler.execute_fetchone(sql, (self._id, ))
-        return inv[0] if inv is not None else inv
+        with TRN:
+            sql = """SELECT investigation_id FROM qiita.investigation_study
```

Comment:
I don't know. To be fair we have not been using investigation, and I think that right now is not needed given that we can have multiple prep templates in a single study.

Investigation was useful in the old database, were we had to create a study for each prep template... but right now we can create a single study and have everything there. And if multiple studies belong to "an investigation" that is a meta-analysis.

I think this is another discussion and this code is duplicating the functionality of the previous code, so if there are no more comments, we should merge this and keep moving forward with the transactionization of the system.